src/test/java/com/wikia/webdriver/testcases/renametool/RenameToolTests.java

```diff
@@ -178,6 +177,12 @@ public void NewUserCreateEditMessageWallAndRenameDone() {
     wall.submitEdition();
     wall.verifyMessageEditText(title, messageEdit, user.getUsername());
 
+    try {
+      Thread.sleep(15000);
+    } catch (InterruptedException e) {
+      e.printStackTrace( );
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1148)